config/initializers/slack.rb

```diff
@@ -1,3 +1,3 @@
-ENV['SLACK_API_TOKEN'] = Settings.slack.api_token
+# ENV['SLACK_API_TOKEN'] = Settings.slack.api_token
```

Comment:
You probably just need to update the readme to specify the need to set this environmental variable. Also probably good just to remove this from the `settings.yml` file.